features/events/edit.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Editing an event from the event show page
+  As a site user
+  In order to be able to update planned activities
+  I would like to be able to edit event details from the event show page
+
+  Background: The following events have been added to the database
+    Given the following organisations exist:
+      | name            | description          | address        | postcode | website       |
+      | Cats Are Us     | Animal Shelter       | 34 pinner road | HA1 4HZ  | http://a.com/ |
+      | Office Primer   | Care for the elderly | 64 pinner road | HA1 4HZ  | http://b.com/ |
+    Given the following events exist:
+      | title               | description       | organisation  | start_date          | end_date            |
+      | Open Source Weekend | Good for everyone | Cats Are Us   | 2030-10-20 10:30:14 | 2030-10-20 17:00:00 |
+      | Lazy Weekend        | Also good         | Office Primer | 2055-02-02 08:00:00 | 2055-02-02 17:00:00 |
+
+  Scenario: As a logged in user, I can navigate to the edit page of an event
```

Comment:
these look good, although a future work might be to make them more declarative and less imperative:

https://wiki.saucelabs.com/display/DOCS/Best+Practice%3A+Imperative+v.+Declarative+Testing+Scenarios